stack/indexer/deb/build_package.sh

```diff
@@ -40,14 +41,16 @@ build_deb() {
     # Copy the necessary files
     cp ${current_path}/builder.sh ${dockerfile_path}
 
-    # Base generation
-    if [ "${future}" == "yes" ];then
-        base_cmd+="--future "
+    if [ "${build_base}" == "yes" ];then
+        # Base generation
+        if [ "${future}" == "yes" ];then
+            base_cmd+="--future "
+        fi
+        if [ "${reference}" ];then
+            base_cmd+="--reference ${reference}"
+        fi
+        ../base/generate_base.sh -s ${outdir} -r ${revision} ${base_cmd}
```

Comment:
I think it's better to use `POSIX =`

`if [ "${build_base}" = "yes" ]`

Reply:
I think it is better to use the operator "==" since with this we keep this operator that is used in other programming languages. Any of the options is valid "=" or "=="